### ISSUE TITLE
Support for reactive forms in ng2

### ISSUE THREAD

**Issue by prassanaj** [atais/ng2-eonasdan-datetimepicker#11]
Im trying to use this directive. I see that you are taking [date] input which makes it not compatible with "from control" of ng2 (reactive form support).  Do you have this support considered already?  If yes, could you please show me how?  I'm attempting to this below.

```
      <div class="input-group">
        <input class="form-control"
               a2e-datetimepicker
               [date]="obj.date"
               [options]="a2eOptions"
               [placeholder]="obj.placeholder"
        />
        <span class="input-group-addon"><span class="fa fa-calendar"></span></span>
      </div>
```

**Comment by atais** [comment]
So now as of 0.1.0 is this ticket still valid? @fetrarij

**Comment by fetrarij** [comment]
This is solved with `0.1.0` .
Check [our example with reactive forms here](https://github.com/atais/ng2-eonasdan-datetimepicker/tree/master/examples/webpack/src/app/reactive-form)
